platform/windows_platform.go

```diff
@@ -28,7 +28,7 @@ import (
 	boshuuid "github.com/cloudfoundry/bosh-utils/uuid"
 )
 
-//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 -o fakes/fake_windows_disk_manager.go . WindowsDiskManager
+//go:generate go run github.com/maxbrunsfeld//v6 -o fakes/fake_windows_disk_manager.go . WindowsDiskManager
```

Comment:
Not sure what happened here, but this change doesn't seem correct. Not sure if it was a typo, or if something weird is going on with `go generate`

Reply:
I guess that was my VSCode. Regenerating everything worked afterwards. Not that familiar with counterfeiter and will believe you and revert this line if you think that is the way to go.

Reply:
Yeah, running `go generate` on my machine without the `counterfeiter` segment results in:
```
cannot find package "." in:
	/Users/jpalermo/workspace/bosh-agent/vendor/github.com/maxbrunsfeld/v6
windows_platform.go:31: running "go": exit status 1

Process finished with exit code 1
```
I also had an `inconsistent vendoring` error when first trying to run `go generate`, you might need to `go mod tidy` & `go mod vendor`.

Reply:
I reverted  the line and did a go mod tidy && vendor. Shoul'd be fixed now. Can you test on your end if it works now?